### DISCUSSION TITLE
Manual/SRPBatcher

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#233] [Notes]
# Manual/SRPBatcher

https://docs.unity3d.com/Manual/SRPBatcher.html

**Comment by mvi** [comment]
As mentioned, SRP Batcher does not work with Material Property Blocks, so if you need to disable SRP Batcher for a renderer you can do it by assigning an empty property block:  
`targetRenderer.SetPropertyBlock(new MaterialPropertyBlock());`
